scripts/actions/reset_default_config

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC STARTING
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source scripts/_common.sh
+source /usr/share/yunohost/helpers
+
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
+
+app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}
+
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
+port=$(ynh_app_setting_get --app=$app --key=port)
+export=$(ynh_app_setting_get --app=$app --key=export)
+language=$(ynh_app_setting_get --app=$app --key=language)
+mypads=$(ynh_app_setting_get --app=$app --key=mypads)
+useldap=$(ynh_app_setting_get --app=$app --key=useldap)
+
+#=================================================
+# SORT OUT THE CONFIG FILE TO HANDLE
+#=================================================
+
+file="$1"
+
+if [ "$file" = "settings.json" ]; then
+	config_file="$final_path/settings.json"
+fi
+
+#=================================================
+# SPECIFIC ACTION
+#=================================================
+# RESET THE CONFIG FILE
+#=================================================
+ynh_script_progression --message="Resetting the config file $config_file..." --weight=3
+
+# Verify the checksum and backup the file if it's different
+ynh_backup_if_checksum_is_different --file="$config_file"
+
+if [ "$file" = "settings.json" ]
+then
+	# Get the default file and overwrite the current config
+	cp /etc/yunohost/apps/$app/conf/settings.json "$config_file"
+
+	# Recreate the default config
+	ynh_replace_string --match_string="__PORT__" --replace_string="$port" --target_file="$final_path/settings.json"
+	if [ "$export" = "abiword" ]
+	then
+		abiword_path=`which abiword`	# Get abiword binary path
+		ynh_replace_string --match_string="\"abiword\" : null" --replace_string="\"abiword\" : \"$abiword_path\"" --target_file="$final_path/settings.json"	# Renseigne l'emplacement de abiword dans la config de etherpad
```

Comment:
> \# Renseigne l'emplacement de abiword dans la config de etherpad

In french? O.o I don't recognize you anymore :'(

Reply:
Probably an old copy past ;)
I'll fix it